python/python/glide/async_commands/core.py

```diff
@@ -2326,6 +2326,33 @@ async def zscore(self, key: str, member: str) -> Optional[float]:
             await self._execute_command(RequestType.ZScore, [key, member]),
         )
 
+    async def zmscore(
+        self,
+        key: str,
+        members: List[str],
+    ) -> List[float]:
+        """
+        Returns the scores associated with the specified `members` in the sorted set stored at `key`.
+
+        See https://redis.io/commands/zmscore for more details.
+
+        Args:
+            key (str): The key of the sorted set.
+            members (List[str]): A list of members in the sorted set.
+
+        Returns:
+            List[float]: A list of scores of the `members`.
+                If a `member` does not exist, the corresponding value in the list will be `None`.
```

Comment:
```suggestion
                If a member does not exist, the corresponding value in the list will be None.
```

we dont write `None` in python, and there's no `member`